api.ts

```diff
@@ -144,6 +144,11 @@ export interface SourceFileConfiguration {
      */
     readonly forcedInclude?: string[];
 
+    /**
+     * Arguments for the compiler.
+     */
+    readonly compilerArgs?: string[];

```

Comment:
super nit.  `compilerArgs` is declared before `compilerPath` in this interface, but after it in the other interface.  Can you put it after `compilerPath` in both cases?

Reply:
Meant to put it after `compilerPath` too.